modules/apps/object/object-admin-rest-impl/rest-openapi.yaml

```diff
@@ -707,7 +709,7 @@ components:
 info:
     description:
         "A Java client JAR is available for use with the group ID 'com.liferay', artifact ID
-        'com.liferay.object.admin.rest.client', and version '1.0.81'."
+        'com.liferay.object.admin.rest.client', and version '1.0.82'."
```

Comment:
was this change made by you or buildRest? 
can you remove and run buildRest to make sure? if so, put this change in the buildRest commit please